### PR TITLE
Add M1 multiplier event planning document

### PR DESCRIPTION
**Pull request by teodutu**

### CODE REVIEW

File: website/docs/events/multiplier-event-1.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Multiplier Event 1: Pre-launch of 1st content
+
+- Date: 27.10.2022
+- Time: 9:00 - 14:00 EEST
+- Place: Room PR 105, Faculty of Automatic Control and Computer Science, POLITEHNICA University of Bucharest
+
+## Initial Presentation: 9:00 - 11:30
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
## Initial Presentation: 9:00 - 11:00
```

File: website/docs/events/multiplier-event-1.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+The team will go over the following topics:
+
+- Overview of the project (Răzvan Deaconescu)
+- Methodology (Răzvan Nițu)
+- Content (Teodor Duțu)
+- Infrastructure (Sergiu Weisz)
+- Community (Liza Babu)
+
+Each talk aims to take around 20 minutes, followed by 10 minutes for questions and discussions.
+
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]: Add a 30 minutes coffee break, 11:00-11:30

File: website/docs/events/multiplier-event-1.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Demo from the Open Education Hub Team: 11:30 - 12:00
+
+In this part we will showcase how to create a repository under the [Open Education Hub orgainsation](https://github.com/open-education-hub/) using our templates and how to contribute to it via pull requests and in accordance to our methodology.
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
In this part we will showcase how to create a repository under the [Open Education Hub organization](https://github.com/open-education-hub/) using our templates and how to contribute to it via pull requests and in accordance to our methodology.
```

File: website/docs/events/multiplier-event-1.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Practical Workshop: 12:00 - 13:00
+
+Participants will create their own repositories within the the [Open Education Hub orgainsation](https://github.com/open-education-hub/) and start developing content in order to familiarise themselves with our workflow.
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
Participants will create their own repositories within the the [Open Education Hub organization](https://github.com/open-education-hub/) and start developing content in order to familiarize themselves with our workflow.
```

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @teodutu . I did a first pass.

File: website/docs/events/multiplier-event-1.md (hunk 1 above)
Comment (teodutu) [reply website/docs/events/multiplier-event-1.md]: You are highly optimistic that we can talk for only ~24 minutes each, questions included. You in particular :)))

File: website/docs/events/multiplier-event-1.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Multiplier Event 1: Pre-launch of 1st content
+
+- Date: 27.10.2022
+- Time: 9:00 - 14:00 EEST
+- Place: Room PR 105, Faculty of Automatic Control and Computer Science, POLITEHNICA University of Bucharest
+
+## Initial Presentation: 9:00 - 11:00
+
+The team and community members will hold presentations regarding the purpose and aims of Open Education Hub, our mission and ideals.
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
The team and community members will host talks regarding the purpose and aims of Open Education Hub.
```

I cut some of it, it seems too much.

File: website/docs/events/multiplier-event-1.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+The team will go over the following topics:
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
We will go over the following topics:
```

File: website/docs/events/multiplier-event-1.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+- Overview of the project (Răzvan Deaconescu)
+- Methodology (Răzvan Nițu)
+- Content (Teodor Duțu)
+- Infrastructure (Sergiu Weisz)
+- Community (Liza Babu)
+
+Each talk aims to take no more than 20 minutes, followed by 5 - 10 minutes for questions and discussions.
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
Each talk aims to take 10-20 minutes, followed by 5 - 10 minutes for questions and discussions.
```

File: website/docs/events/multiplier-event-1.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+## Coffee Break: 11:00 - 11:30
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
## Coffee Break: 10:30 - 11:00
```

File: website/docs/events/multiplier-event-1.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+This will be an opportunity for more in-depth and informal discussions regarding the topics presented so far.
+
+## Demo from the Open Education Hub Team: 11:30 - 12:00
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
## Demo of Open Education Hub: 11:00 - 11:30
```

File: website/docs/events/multiplier-event-1.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+In this part we will showcase how to create a repository under the [Open Education Hub organization](https://github.com/open-education-hub/) using our templates and how to contribute to it via pull requests and in accordance to our methodology.
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
In this part we will showcase how to create a repository under the [Open Education Hub organization](https://github.com/open-education-hub/) using our templates and how to contribute to it via pull requests and in accordance to [the Open Education Hub methodology](https://github.com/open-education-hub/methodology).
```

File: website/docs/events/multiplier-event-1.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+## Practical Workshop: 12:00 - 13:00
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
## Practical Workshop: 11:30 - 13:00
```

File: website/docs/events/multiplier-event-1.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+Participants will create their own repositories within the the [Open Education Hub organization](https://github.com/open-education-hub/) and start developing content in order to familiarize themselves with our workflow.
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]: I wouldn't create them in the organization. They can be "hosted" by Open Education Hub once they pass certain criteria (that we have yet to write down).

File: website/docs/events/multiplier-event-1.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+
+The participants will be required to prepare a few ideas for what content create in advance: general theme, main points they would .
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
We ask participants to prepare a few ideas about the repository they want to create.
```

File: website/docs/events/multiplier-event-1.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+They will have to think about topics such as:
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
Please consider topics such as:
```

File: website/docs/events/multiplier-event-1.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+
+- general theme and subject of their repositories
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
- general theme and subject of your repository
```

File: website/docs/events/multiplier-event-1.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+- the main ideas / points they would like to deliver to the future students learning the material created by them
```
Comment (razvand) [inline website/docs/events/multiplier-event-1.md]:
```suggestion
- main ideas / points you would like to deliver to future students
```

File: website/docs/events/multiplier-event-1.md (hunk 12 above)
Comment (razvand) [reply website/docs/events/multiplier-event-1.md]: I would reword this as participants are "encouraged". And tell they would require a laptop. (Wireless) Internet access will be provided.

But participants can also just help others, or engage the Open Education Hub team. I wouldn't "require" them to create content / repositories.

I think the most important part is for them to reproduce the build / deployment of the Operating Systems class. That I think is the most important part. The creation of another repository I would mark as optional.

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @teodutu . See my updates / clarifications.

File: website/docs/events/multiplier-event-1.md (hunk 8 above)
Comment (teodutu) [reply website/docs/events/multiplier-event-1.md]: 10:30 seems like wishful thinking, but let's try it.

File: website/docs/events/multiplier-event-1/description.md [hunk 17]
```diff
@@ -0,0 +1,50 @@
+# Description
+
+- Date: 27.10.2022
+- Time: 9:00 - 14:00 EEST
+- Place: Room PR 105, Faculty of Automatic Control and Computer Science, POLITEHNICA University of Bucharest
+
+## Initial Presentation: 9:00 - 11:00
```
Comment (razvand) [inline website/docs/events/multiplier-event-1/description.md]:
```suggestion
## Initial Presentation: 9:00 - 10:30
```

**Review by razvand — CHANGES_REQUESTED** [review]
@teodutu , see my small comment. Please remove the slides skeleton in another PR. Let's merge the event PR first, then we work on the other PR for slides.

File: website/docs/events/multiplier-event-1/_category_.json [hunk 18]
```diff
@@ -0,0 +1,5 @@
+{
+  "label": "Multiplier Event 1: Pre-launch of 1st Content",
+  "position": 1
+}
+
```
Comment (razvand) [inline website/docs/events/multiplier-event-1/_category_.json]:
```suggestion
```

File: website/.gitignore [hunk 19]
```diff
@@ -3,3 +3,5 @@
 
 .docusaurus
 .cache-loader
+
+_site/
```
Comment (razvand) [inline website/.gitignore]: The `.gitignore` file update should be part of a different commit.

**Review by razvand — CHANGES_REQUESTED** [review]
See comments.

**Review by razvand — APPROVED** [review]
All good. Thanks, @teodutu
